cpp/ql/test/library-tests/ir/ssa/ssa.cpp

```diff
@@ -102,3 +102,8 @@ void MayPartiallyOverlapEscaped(int x, int y) {
   Point b = a;
   Escape(&a);
 }
+
```

Comment:
I don't think this test case needs to be in the SSA tests. I was intending the SSA tests to be just for test cases specific to use/def chaining, Phi, and Chi insertion, rather than an exhaustive test of SSA construction on all C++ constructs.